_docs_operate/setup-with-helm-charts.md

```diff
@@ -130,6 +132,11 @@ config:
 pod:
   connector:
     environment:
+      - name: transportLibrary__baseUrl
+        alueFrom:
+          secretKeyRef:
+            name: base-url
+            key: VALUE
       - name: transportLibrary__platformClientId
         alueFrom:
```

Comment:
```suggestion
        valueFrom:
```

Reply:
good catch! I copied from the wrong place!